docs/reference_guide.md

```diff
@@ -1241,7 +1248,23 @@ block I/O created by 28941
 Examples in situ:
 [search /tools](https://github.com/iovisor/bpftrace/search?q=tracepoint%3A+path%3Atools&type=Code)
 
-## 6. `tracepoint`: Static Tracing, Kernel-Level Arguments
+```
+# bpftrace -e 'rawtracepoint:block_rq_insert { printf("block I/O created by %d\n", tid); }'
+Attaching 1 probe...
+block I/O created by 189126
+[...]
+```
+
+## 6. `tracepoint`/`rawtracepoint`: Static Tracing, Kernel-Level Arguments
+
+The bpf ctx parameters of tracepoint and rawtracepoint are different, rawtracepoint is lower level, and tracepoint is processed by the kernel on the basis of the parameters of rawtracepoint.
```

Comment:
I understand what you mean here but probably only b/c I already know what raw tracepoints are. This might be too ambiguous for people new to it. Perhaps something like:

```suggestion
`tracepoint` and `rawtracepoint` are nearly identical in terms of functionality. The only difference is in the program context. `rawtracepoint` offers raw arguments to the tracepoint while `tracepoint` applies further processing to the raw arguments. The additional processing is defined inside the kernel.
```

Reply:
I'd still consider separating raw tracepoints section from tracepoints (just keep them next to each other). They are quite different from the usage point of view (raw tracepoints don't have categories, arguments are accessed differently). But that's just an idea, not insisting on it.

Reply:
I'd still consider separating raw tracepoints section from tracepoints (just keep them next to each other). They are quite different from the usage point of view (raw tracepoints don't have categories, arguments are accessed differently). But that's just an idea, not insisting on it.

Reply:
But I really like your rewording here, no problem using that.

Reply:
I don't have a strong opinion on separate or together. Separate is probably cleaner from a docs perspective.

Reply:
Will do. I'll revise it next week and add test case, glad to get your suggestions.